src/wp-admin/includes/class-wp-site-health.php

```diff
@@ -764,33 +764,62 @@ public function get_test_php_version() {
 		if ( $response['is_supported'] ) {
 			$result['label'] = sprintf(
 				/* translators: %s: The server PHP version. */
-				__( 'Your site is running an older version of PHP (%s)' ),
+				__( 'Your site is running on an older version of PHP (%s)' ),
 				PHP_VERSION
 			);
 			$result['status'] = 'recommended';
 
 			return $result;
 		}
 
+		// The PHP version is still receiving security fixes, but is lower than
+		// the expected minimum version that will be required by WordPress in the near future.
```

Comment:
```suggestion
		/*
		 * The PHP version is still receiving security fixes, but is lower than
		 * the expected minimum version that will be required by WordPress in the near future.
		 */
```
[Ref](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#5-2-multi-line-comments).

Reply:
Hmm, it's probably not that important but this:
```
/*
 * Line 1
 * Line 2
 * Line 3
 */
```
is too close to this:
```
/**
 * Line 1
 * Line 2
 * Line 3
 */
```

For that reason I think it is the inferior way to do multi-line comments. I can see it in the doc standards, but it was never a part of the initial coding style. Not sure who added it there and why. Initially comments styling was open, just like in the JS standards (Prettier).

Think the standards would have to be reviewed as I couldn't find where it was decided to amend them and add that restriction.

Reply:
From what I can tell, so far at least, the format has been documented since before [this commit](https://github.com/WordPress/wpcs-docs/commit/2568ca56640ba5f85c09e77e4d8fca05362cae57) in 2018. Before that, it was in the Handbook.

Just to check: Are you saying that you'll implement the current standards but you'd like Core to discuss the multi-line comment format, or are you saying you won't implement the current standards until Core discusses the multi-line comment format?

Genuine question as I wasn't sure after reading.

Reply:
The coding standards/style guidelines have existed since 2007-2008 (I think). There wasn't such restriction there until at least 2014-2015. I'll try to do some digging to see when it was added.

> Are you saying that you'll implement the current standards...
Sure, although they make the comments styling worse.

Reply:
Do ping me when you find out! :slightly_smiling_face: